packages/api-gateway-service/.env.example

```diff
@@ -15,8 +15,13 @@ COOKIE_PARSER=
 # INTERNAL_IDP=
 
 ## Dependent Microservices
-## User Group
-# USER_SERVICE_SERVICE_HOST="localhost:8080"
-# HOME_SERVICE_SERVICE_HOST="localhost:8081"
-# FEEDBACK_SERVICE_SERVICE_HOST="localhost:8082"
-# NOTIFICATIONS_SERVICE_SERVICE_HOST="localhost:8083"
+# USER_SERVICE_SERVICE_HOST="localhost"
+# HOME_SERVICE_SERVICE_HOST="localhost"
+# FEEDBACK_SERVICE_SERVICE_HOST="localhost"
+# NOTIFICATIONS_SERVICE_SERVICE_HOST="localhost"
+
+## Service Ports
+# USER_SERVICE_SERVICE_PORT="8080"
+# HOME_SERVICE_SERVICE_PORT="8081"
+# FEEDBACK_SERVICE_SERVICE_PORT="8082"
+# NOTIFICATIONS_SERVICE_SERVICE_PORT="8082"
```

Comment:
`8080` will be the API Gateway, so the microservices need to be on other ports.
Also, use the ports configured in the docker-compose so it's easier and compatible with docker-compose.

```suggestion
# HOME_SERVICE_SERVICE_PORT="8081"
# FEEDBACK_SERVICE_SERVICE_PORT="8082"
# USER_SERVICE_SERVICE_PORT="8083"
# NOTIFICATIONS_SERVICE_SERVICE_PORT="8084"
```